test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -644,5 +645,18 @@ public void TestLdaCommandLine()
         {
             Assert.Equal(Maml.Main(new[] { @"showschema loader=Text{col=A:R4:0-10} xf=lda{col=B:A} in=f:\2.txt" }), (int)0);
         }
+
+        [Fact]
+        public void TestTextFeaturizerBackCompat()
+        {
+            var modelPath = GetDataPath("backcompat", "SentimentModel.zip");
+            var model = ML.Model.Load(modelPath, out var inputSchema);
```

Comment:
Do we want to use the model as well? Just to make sure everything is hooked up correctly?